### PR TITLE
fix: Naming in Protobuf

### PR DESCRIPTION
**Pull request by meerhat**
# Description

Fix for https://github.com/zeta-chain/node/issues/950
Changed the .proto file containing receiver_chainId to receiver_chain_id.

## Summary by CodeRabbit

- **Chores**
	- Updated property name from `receiver_chainId` to `receiver_chain_id` in the OpenAPI specification for consistency.
	- Renamed field `receiver_chainId` to `receiver_chain_id` in the `OutboundParams` message to align with naming conventions. 

These changes enhance clarity and maintain consistency across the documentation and protocol definitions.

### CODE REVIEW

File: docs/openapi/openapi.swagger.yaml [hunk 1]
```diff
@@ -57280,7 +57280,7 @@ definitions:
     properties:
       receiver:
         type: string
-      receiver_chainId:
+      receiver_chain_id:
```
Comment (swift1337) [inline docs/openapi/openapi.swagger.yaml:57283]: I'm pretty sure this will break existing indexers.

Comment (meerhat) [reply docs/openapi/openapi.swagger.yaml]: I've deleted this from the changed files

**Review by gartnera — CHANGES_REQUESTED** [review]
This is a breaking change and needs to be carefully considered.

**Comment by lumtis** [comment]
Hi @meerhat thanks for the contribution

However, as mentioned in the conversation, this will bring a breaking change. We're looking for now to avoid having breaking changes in the coming upgrades, we will so close this PR for the time being.

Feel free to continue the conversation on the associated issue. Thanks.

**Pull request closed without merging**